.github/workflows/Build.yml

```diff
@@ -210,16 +234,15 @@ jobs:
         env:
           UPDATE_BRANCH_NAME: ${{ github.ref_name }}
         run: |
+          git log --oneline -10
           if [ -n "$(git status --porcelain)" ]; then
+            git log --oneline -10
```

Comment:
Would this be necessary since it's already run prior to the conditional?

Reply:
The git log lines are for debugging. I will remove most of those.